ProcessLib/AbstractJacobianAssembler.h

```diff
@@ -41,6 +42,8 @@ class AbstractJacobianAssembler
         std::vector<double>& /*local_Jac_data*/,
         LocalCoupledSolutions const& /*coupled_solutions*/)
     {
+        // TODO make pure virtual.
```

Comment:
Please add a short why, and maybe how (if not obvious).

Reply:
Why ... because implementing this can be easily forgotten in derived classes. E.g., it's not implemented in the central differences assembler if I remember correctly.
How ... obviously: make it pure virtual and add the missing implementations.

Do you really want to have these comments in the code?